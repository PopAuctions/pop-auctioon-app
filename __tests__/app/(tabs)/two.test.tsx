import * as React from 'react';
import renderer from 'react-test-renderer';
import TabTwoScreen from '../../../app/(tabs)/two';

describe('TabTwoScreen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TabTwoScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
