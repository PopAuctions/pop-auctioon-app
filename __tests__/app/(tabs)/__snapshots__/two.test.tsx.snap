// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabTwoScreen renders correctly 1`] = `null`;
